include/z64save.h

```diff
@@ -644,31 +644,55 @@ typedef enum {
  */
 
 // 0x00-0x0F
-#define EVENTINF_0X_INDEX 0
-#define EVENTINF_00_SHIFT 0
-#define EVENTINF_01_SHIFT 1
-#define EVENTINF_02_SHIFT 2
-#define EVENTINF_03_SHIFT 3
-#define EVENTINF_04_SHIFT 4
-#define EVENTINF_05_SHIFT 5
-#define EVENTINF_06_SHIFT 6
-#define EVENTINF_08_SHIFT 8
-#define EVENTINF_0A_SHIFT 10
-#define EVENTINF_0F_SHIFT 15
-#define EVENTINF_00_MASK (1 << EVENTINF_00_SHIFT)
-#define EVENTINF_01_MASK (1 << EVENTINF_01_SHIFT)
-#define EVENTINF_02_MASK (1 << EVENTINF_02_SHIFT)
-#define EVENTINF_03_MASK (1 << EVENTINF_03_SHIFT)
-#define EVENTINF_04_MASK (1 << EVENTINF_04_SHIFT)
-#define EVENTINF_05_MASK (1 << EVENTINF_05_SHIFT)
-#define EVENTINF_06_MASK (1 << EVENTINF_06_SHIFT)
-#define EVENTINF_0F_MASK (1 << EVENTINF_0F_SHIFT)
-#define EVENTINF_04 ((EVENTINF_0X_INDEX << 4) | EVENTINF_04_SHIFT)
-#define EVENTINF_05 ((EVENTINF_0X_INDEX << 4) | EVENTINF_05_SHIFT)
-#define EVENTINF_06 ((EVENTINF_0X_INDEX << 4) | EVENTINF_06_SHIFT)
-#define EVENTINF_08 ((EVENTINF_0X_INDEX << 4) | EVENTINF_08_SHIFT)
-#define EVENTINF_0A ((EVENTINF_0X_INDEX << 4) | EVENTINF_0A_SHIFT)
-#define EVENTINF_0F ((EVENTINF_0X_INDEX << 4) | EVENTINF_0F_SHIFT)
+// horses related
+#define EVENTINF_HORSES_INDEX 0
+#define EVENTINF_HORSES_STATE_SHIFT 0
+#define EVENTINF_HORSES_HORSETYPE_SHIFT 4
+#define EVENTINF_HORSES_05_SHIFT 5
+#define EVENTINF_HORSES_06_SHIFT 6
+#define EVENTINF_HORSES_08_SHIFT 8
+#define EVENTINF_HORSES_0A_SHIFT 10
+#define EVENTINF_HORSES_0F_SHIFT 15 // unused?
+#define EVENTINF_HORSES_STATE_MASK (0xF << EVENTINF_HORSES_STATE_SHIFT)
+#define EVENTINF_HORSES_HORSETYPE_MASK (1 << EVENTINF_HORSES_HORSETYPE_SHIFT)
+#define EVENTINF_HORSES_05_MASK (1 << EVENTINF_HORSES_05_SHIFT)
+#define EVENTINF_HORSES_06_MASK (1 << EVENTINF_HORSES_06_SHIFT)
+#define EVENTINF_HORSES_0F_MASK (1 << EVENTINF_HORSES_0F_SHIFT)
+#define EVENTINF_HORSES_05 ((EVENTINF_HORSES_INDEX << 4) | EVENTINF_HORSES_05_SHIFT)
+#define EVENTINF_HORSES_06 ((EVENTINF_HORSES_INDEX << 4) | EVENTINF_HORSES_06_SHIFT)
+#define EVENTINF_HORSES_08 ((EVENTINF_HORSES_INDEX << 4) | EVENTINF_HORSES_08_SHIFT)
+#define EVENTINF_HORSES_0A ((EVENTINF_HORSES_INDEX << 4) | EVENTINF_HORSES_0A_SHIFT)
+
+typedef enum {
+    /* 0 */ EVENTINF_HORSES_STATE_0,
+    /* 1 */ EVENTINF_HORSES_STATE_1,
+    /* 2 */ EVENTINF_HORSES_STATE_2,
+    /* 3 */ EVENTINF_HORSES_STATE_3,
+    /* 4 */ EVENTINF_HORSES_STATE_4,
+    /* 5 */ EVENTINF_HORSES_STATE_5,
+    /* 6 */ EVENTINF_HORSES_STATE_6,
+    /* 7 */ EVENTINF_HORSES_STATE_7
+} EventInfHorsesState;
+
+// "InRaceSeq"
+#define GET_EVENTINF_HORSES_STATE() \
```

Comment:
I don't really like how these look with `()`.  I feel like it will make some people think its a function.

Reply:
it essentially behaves as one, its a natural way for getters and setters to look

Reply:
it was specifically requested by roman here https://github.com/zeldaret/oot/pull/1214#discussion_r867361339
and I agree with the choice, it looks better this way imo

Reply:
I am going to merge this now as it has enough approvals to do so. But if anyone else shares this same sentiment and it needs further discussion, feel free to do so